tests/hmm/plotting.py

```diff
@@ -1056,10 +1082,10 @@ def PrintDatacard(categories, dict_procs, era, event_counts, filenames, ofname):
                                 plot_args_shape_syst += [(
                                     histos, hdata, mc_samp, analysis,
                                     var, "nominal", weight_xs, int_lumi, outdir, era, unc)]
-        rets = list(pool.map(make_pdf_plot, plot_args))
+        #rets = list(pool.map(make_pdf_plot, plot_args))
```

Comment:
Perhaps this can be uncommented as well?

Reply:
done